bundles/org.openhab.binding.hue/src/main/resources/OH-INF/i18n/hue.properties

```
@@ -148,6 +148,7 @@ config-status.error.missing-ip-address-configuration = No IP address for the Hue
 # thing status descriptions
 
 offline.communication-error = An unexpected exception occurred during execution.
+offline.conf-error-https-connection = HTTPS secure connection failed. Please check your configuration settings (network address, protocol, port, type of certificate) and change configuration to HTTP when using a V1 bridge.
```

Single comment, would prefer if @cweitkamp could join as reviewer:
```suggestion
offline.conf-error-https-connection = HTTPS secure connection failed. Please check your configuration settings (network address, protocol, port, type of certificate) and change protocol to http when using a V1 bridge.
```

---

Done